firestore/tests/system.py

```diff
@@ -409,7 +409,13 @@ def test_collection_add(client, cleanup):
     cleanup(document_ref1)
     snapshot1 = document_ref1.get()
     assert snapshot1.to_dict() == data1
-    assert snapshot1.create_time == update_time1
+    # NB: We used to assert here that the snapshot create_time was the document
+    # update time as returned by add(), but now under the hood, to allow for
+    # transform values passed into add, the document is created, then a set()
+    # is issued against the document later; the value returned by add is the
+    # result of the latter.  We don't have access to the create time,
+    # so we don't have a value to make this assertion with.
+    # assert snapshot1.create_time == update_time1
```

Comment:
Could that assertion still be useful, but as `assert snapshot.create_time < update_time`?